acceptance-tests/upgrade/update_and_upgrade_mysql_test.go

```diff
@@ -23,7 +23,7 @@ var _ = Describe("UpgradeMYSQLTest", Label("mysql"), func() {
 			defer serviceBroker.Delete()
 
 			By("creating a service instance")
-			serviceInstance := services.CreateInstance("csb-google-mysql", "small", services.WithBroker(serviceBroker))
+			serviceInstance := services.CreateInstance("csb-google-mysql", "default", services.WithBroker(serviceBroker))
```

Comment:
the released version doesn't have this plan. The instance needs to be created with the small plan, a plan that mimics the small plan needs to be set up in the environment variables for the latest version (latestEnv) and  after upgrading the broker the instance should be changed to the new plan with `cf update-service`

Reply:
Thank you very much @pivotal-marcela-campo.